lab.js

```diff
@@ -12,29 +12,32 @@ var applesObj = {
 
 // a. Set eveAppleCount equal to the number of apples that Eve has
 
-let eveAppleCount /* = Your code here */
+// let eveAppleCount = 4
 
 // assert(eveAppleCount === 4)
 
 // b. Change the number of apples that Adam  has to 4
 
-// Your code here
+// applesObj.Adam = 4 
 
-//assert(applesObj["Adam"] === 4)
+// assert(applesObj["Adam"] === 4)
 
 // c. Set calAndDanAppleCount equal to how many apples Cal and Dan have combined
 
-let calAndDanAppleCount /*= Your code here */
+// let calAndDanAppleCount = 8
```

Comment:
we were looking for a more programatic answer (one that will work even when the values change)